### PR TITLE
QRadar IPs commands enhancement

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40246

## Description
Exposing the IPs APIs calls used in QRadar v3 integration for user commands

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
@David-BMS FYI amigo

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 1]
```diff
@@ -2350,3 +2350,215 @@ There is no context output for this command.
...
+            "LocalDestinationAddressIDs": [
+                1,
+                2,
+                3,
+                4,
+                5
+            ],
+            "Magnitude": 0,
+            "Network": "Net-10-172-192.Net_172_16_0_0",
+            "OffenseIDs": [
```
Comment (DeanArbel) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:2410]: nice

**Review by DeanArbel — APPROVED** [review]
good job

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml [hunk 2]
```diff
@@ -2814,6 +2812,124 @@ script:
...
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      description: 'If used, will filter all fields except for the specified ones.
+        Use this parameter to specify which fields you would like to get back in the
+        response. Fields that are not named are excluded. Specify subfields in brackets
+        and multiple fields in the same object separated by commas. The filter uses
+        QRadar''s field names, for reference, consult: https://www.ibm.com/docs/en/qradar-common?topic=endpoints-get-siemlocal-destination-addresses'
```
Comment (roysagi) [inline Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml:2827]:
```suggestion
      description: 'If used, will filter all fields except for the ones specified.
        Use this argument to specify which fields should be returned in the
        response. Fields that are not named are excluded. Specify subfields in brackets
        and multiple fields in the same object separated by commas. The filter uses
        QRadar's field names, for reference, consult: https://www.ibm.com/docs/en/qradar-common?topic=endpoints-get-siemlocal-destination-addresses'
```

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml [hunk 3]
```diff
@@ -2814,6 +2812,124 @@ script:
...
```
Comment (roysagi) [inline Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml:2886]: same notes as above

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 4]
```diff
@@ -2350,3 +2350,215 @@ There is no context output for this command.
...
+                33,
+                34,
+                35,
+                36,
+                37,
+                38,
+                39,
+                40,
+                41,
+                42
```
Comment (roysagi) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:2446]: probably should cut down the numbers just for it to be a bit more readable

File: Packs/QRadar/ReleaseNotes/2_0_21.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Added new commands: `qradar-ips-source-get` and `qradar-ips-local-destination-get`.
```
Comment (roysagi) [inline Packs/QRadar/ReleaseNotes/2_0_21.md:4]:
```suggestion
- Added new commands: ***qradar-ips-source-get*** and ***qradar-ips-local-destination-get***.
```

**Comment by roysagi** [comment]
@tomneeman151293 - please attend to my doc notes before merging

**Pull request merged**